aten/src/ATen/native/native_functions.yaml

```diff
@@ -1546,6 +1546,12 @@
 - func: rowwise_prune(Tensor weight, Tensor mask, ScalarType compressed_indices_dtype) -> (Tensor, Tensor)
   use_c10_dispatcher: full
 
+# row_stack is the alias of vstack
+- func: row_stack(Tensor[] tensors) -> Tensor
+  use_c10_dispatcher: full
+
```

Comment:
Dispatch was recently updated (see https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/README.md). As a result, I think this (and row_stack.out, below) should get a new "Math" dispatch key:

```
Dispatch:
  Math: row_stack
```

Reply:
Thanks for the kind tip, the `Math` dispatch keys are added for both `row_stack` and `row_stack.out`.